src/types/fpti.js

```diff
@@ -106,3 +106,81 @@ export type FptiVariables = {|
   // Timestamp relative to user
   g : Date
 |};
+
+/* Workaround for the sake of supporting the 'legacy' store cash implementation. 'legacy'
```

Comment:
this comment is awesome, i would try to generalize it though, supports `legacy` pp shopping fpti events